optimum/pipelines.py

```diff
@@ -119,13 +168,14 @@ def pipeline(
     accelerator: Optional[str] = "ort",
     **kwargs,
 ) -> Pipeline:
-
     targeted_task = "translation" if task.startswith("translation") else task
+    if accelerator == "bettertransformer":
+        from transformers.pipelines import SUPPORTED_TASKS
 
     if targeted_task not in list(SUPPORTED_TASKS.keys()):
         raise ValueError(f"Task {targeted_task} is not supported. Supported tasks are { list(SUPPORTED_TASKS.keys())}")
 
-    if accelerator != "ort":
+    if accelerator not in ["ort", "bettertransformer"]:
```

Comment:
Maybe create a mapping between accelerator anmes and loading functions, then use both to check if the accelerator is in the mapping and to retrieve the loading function later on (instead of `if/else` statements)

Reply:
Nice! Fixed in b76e9c187e82e80e4272d73b902bf76ad4a0852b